velox/docs/functions/spark/math.rst

```diff
@@ -80,6 +80,16 @@ Mathematical Functions
         SELECT 2L / 2L; -- 1.0
         SELECT 3 / 0; -- NULL
 
+.. spark:function:: long_divide(x, y) -> long
```

Comment:
Generally, the function name is the one registered. Curious why not using "div", which is consistent with Spark.